gradle.properties

```diff
@@ -5,10 +5,10 @@
 
 # Version number to use for the generated SDK
 # Note: these must always be full 3-segment semver versions
-aws.sdk.version=0.0.24-alpha
+aws.sdk.version=0.0.25-alpha
 
 # Version number to use for the generated runtime crates
-smithy.rs.runtime.crate.version=0.27.0-alpha.2
+smithy.rs.runtime.crate.version=0.28.0-alpha
```

Comment:
should we start at `-alpha.0` instead of going from `-alpha` to `-alpha.0`?

Reply:
We decided to go with `-alpha` in an offline conversation.